### PR TITLE
Added everything)

### PR DESCRIPTION
**Pull request by igorslepenkov**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,26 +4,70 @@
...
+                <ul>
+                    <li><a href="">Home</a></li>
+                    <li><a href="">Features</a></li>
+                    <li><a href="">Pricing</a></li>
+                    <li><a href="">Blog</a></li>
+                </ul> 
+            </nav>
+
+            <div class="header__button__wrap">
+                <button type="button">Get started</button>
```
Comment (shuliakleonid) [inline index.html:28]: you should write naming class block__element-modifyer

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+.header__button__wrap button {
+    height: 3.8em;
+    width: 12.5em;
+    border-radius: 10px;
+    border: none;
+    background-color: #ff7143;
+    color: rgb(255, 255, 255);
+    font-size: 17px;
```
Comment (shuliakleonid) [inline styles.css:90]: if you start in HEX color, why you write in RGB

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+.get-started {
+    width: 13.2em;
+    height: 3.8em;
+    border: none;
+    border-radius: 0.63em;
+    background-color: #5454D4;
+    color: rgb(255, 255, 255);
+    margin-right: 2.5em;
```
Comment (shuliakleonid) [inline styles.css:125]: color

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    width: 3.8em;
+    height: 3.8em;
+    border-radius: 50%;
+    border: 0.07em solid #e7e7e7;
+    background-color: rgb(255, 255, 255);
+}
+
+.arrow-down:hover {
+    background-color: rgb(226, 234, 255);
+}
```
Comment (shuliakleonid) [inline styles.css:144]: color

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

**Comment by shuliakleonid** [comment]
## Good Work!

File: index.html (hunk 1 above)
Comment (igorslepenkov) [reply index.html:28]: Fixed

File: styles.css (hunk 2 above)
Comment (igorslepenkov) [reply styles.css:90]: Fixed

File: styles.css (hunk 3 above)
Comment (igorslepenkov) [reply styles.css:125]: Fixed

File: styles.css (hunk 4 above)
Comment (igorslepenkov) [reply styles.css:144]: Fixed

**Review by egkirpichev — APPROVED** [review]
well done!

**Review by AlexisKozlov — APPROVED** [review]

**Review by shuliakleonid — APPROVED** [review]

**Pull request merged**